.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint",
+    "prettier"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
+  "rules": {
+    "no-console": 1,       // Means warning
+    "prettier/prettier": 1 // Warning for now, until we get this cleaned up
+  }
+}
```

Comment:
I think we need to remove eslint config from package.json in this case

Reply:
Good point 👍 

May I ask what the rationale behind these rules was? I'm wondering if I should move them over or not.

```
    "no-undefined": "off",
    "no-unused-vars": "off",
```

Reply:
![image](https://user-images.githubusercontent.com/75169/98028864-64d9ab80-1e17-11eb-8c17-7055ac9f41ce.png)

I'm not the right person for this question) I think these rules should not be disabled, they are helpful